app/src/main/java/nextstep/shoppingcart/ui/product/list/ProductListScreen.kt

```diff
@@ -0,0 +1,85 @@
+package nextstep.shoppingcart.ui.product.list
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ShoppingCart
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import nextstep.shoppingcart.R
+import nextstep.shoppingcart.ui.model.dummyProducts
+import nextstep.shoppingcart.ui.product.list.component.ProductListItem
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ProductListScreen() {
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
+            CenterAlignedTopAppBar(
+                title = { Text(text = stringResource(id = R.string.product_list)) },
+                actions = {
+                    Icon(
+                        imageVector = Icons.Filled.ShoppingCart,
+                        contentDescription = "shoppingCartIcon",
+                        tint = Color.Black,
+                        modifier = Modifier
+                            .padding(4.dp)
+                            .size(48.dp)
+                            .padding(8.dp),
+                    )
+                },
+            )
+        },
+    ) { innerPadding ->
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding),
+        ) {
+            ProductListContent()
+        }
+    }
+}
+
+@Composable
+private fun ProductListContent(
+    modifier: Modifier = Modifier,
+) {
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(2),
+        contentPadding = PaddingValues(horizontal = 18.dp, vertical = 13.dp),
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        horizontalArrangement = Arrangement.spacedBy(13.dp),
+        modifier = modifier,
+    ) {
+        items(dummyProducts) {
```

Comment:
지금의 코드와 아래와 같이 변경했을 때의 코드는 어떤 차이가 있을까요?

```suggestion
@Composable
private fun ProductListContent(
    dummyProducts: List<Product>
    modifier: Modifier = Modifier,
) {
    LazyVerticalGrid(
        columns = GridCells.Fixed(2),
        contentPadding = PaddingValues(horizontal = 18.dp, vertical = 13.dp),
        verticalArrangement = Arrangement.spacedBy(20.dp),
        horizontalArrangement = Arrangement.spacedBy(13.dp),
        modifier = modifier,
    ) {
        items(dummyProducts) {
```

힌트: 테스트 관점, Preview관점에서 고려해보세요!

Reply:
추후에 테스트코드 작성할 때에 외부에서 `products` 주입하여 테스트에 더 용이할 거 같습니다